launcher.py

```diff
@@ -0,0 +1,173 @@
+    '''
+    def __init__(self, git_commit_hash: str, platform: str):
+        self.git_commit_hash = git_commit_hash
+        self.platform = platform
+
+    def start(self):
+        ''' Start the launch process for the Childchain service
+        '''
+        logging.info('Service type to launch is Plasma Childchain')
+        logging.info(
+            'Starting launch process for build {}'.format(self.git_commit_hash)
+        )
+        self.ethereum_client = check_ethereum_client(self.platform)
+        logging.info('Ethereum client is {}'.format(self.ethereum_client))
+        if self.compile_application() is False:
+            logging.critical('Could not compile application. Exiting.')
+            sys.exit(1)
+        if self.deploy_plasma_contract() is False:
+            logging.critical('Contract not deployed. Exiting.')
+            sys.exit(1)
+        if self.initialise_childchain_database() is False:
+            logging.critical('Could not initialise database. Exiting.')
+            sys.exit(1)
+        self.start_childchain_service()
+
+    def compile_application(self) -> bool:
+        ''' Execute a mix compile
+        '''
+        result = subprocess.run(['mix', 'compile'], stdout=subprocess.DEVNULL)
+        if result.returncode == 0:
+            logging.info('Elixir mix compile successful')
+            return True
+        return False
+
+    def deploy_plasma_contract(self) -> bool:
+        ''' Deploy the smart contract and populate the ~/config.exs file
+        '''
+        result = subprocess.run(
+            ['./deploy_and_populate.sh'],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT
+        )
+        if result.returncode == 0:
+            logging.info(
+                'Smart contract deployed and authority address populated'
+            )
+            return True
+        logging.critical(
+            'Could not deploy the smart contract. Elixir output: {}'.format(
+                result.stdout
+            )
+        )
+        return False
+
+    def initialise_childchain_database(self) -> bool:
+        ''' Initialise the childchian database (chain data store)
+        '''
+        result = subprocess.run(
+            ["mix", "run", "--no-start", "-e", "'OMG.DB.init()'"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+        )
+        if result.returncode == 0:
+            logging.info('Childchain database initialised')
+            return True
+        logging.critical(
+            'Could not initialise the database. Error: {}'.format(
+                result.stdout
+            )
+        )
+        return False
+
+    def start_childchain_service(self):
+        ''' Start the childchain service
+        '''
+        os.chdir(os.getcwd() + '/apps/omg_api')
+        process = subprocess.Popen(
+            [
+                "/usr/local/bin/iex ", "-S ", "mix", "run", "--config",
```

Comment:
pls be aware that this starts the REPL (`iex`)

Reply:
From https://github.com/omisego/elixir-omg#setting-up-a-child-chain-server-a-developer-environment, `iex -S mix run --config ~/config.exs` is the command to start the chain?